doc/release-notes.md

```diff
@@ -135,6 +135,10 @@ Updated settings
   in future releases. Refer to the help of the affected settings `-whitebind`
   and `-whitelist` for more details. (#19191)
 
+- Netmasks that contain 1-bits after 0-bits (the 1-bits are not contiguous on
```

Comment:
Perhaps point out where these are used. I think the `-rpcallowip` parameter, and the `setban` RPC are the only ones?

Reply:
Will do so in a followup.